types/react-tag-input/index.d.ts

```diff
@@ -45,6 +45,7 @@ export interface ReactTagsProps {
 
     inline?: boolean | undefined;
     inputFieldPosition?: 'top' | 'bottom' | 'inline' | undefined;
+    inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
```

Comment:
```suggestion
    inputProps?: React.InputHTMLAttributes<HTMLInputElement> | undefined;
```
(to stay in sync with other, this is optional in props, supports 4.4 exact optional property types)